### PR TITLE
[Table] Add toggle for custom column headers to preview

### PR DESCRIPTION
**Pull request by gscshoyru**
#### Changes proposed in this pull request:

Add toggle for custom headers, which was missing

### CODE REVIEW

File: packages/table/preview/mutableTable.tsx [hunk 1]
```diff
@@ -603,6 +603,7 @@ export class MutableTable extends React.Component<{}, IMutableTableState> {
                 {this.renderSwitch("Loading state", "showColumnHeadersLoading")}
                 {this.renderSwitch("Menus", "showColumnMenus")}
                 <h6>Interactions</h6>
+                {this.renderSwitch("Custom Headers", "enableColumnCustomHeaders")}
```
Comment (cmslewis) [inline packages/table/preview/mutableTable.tsx]: I'd say this belongs in the `Display` section. 🤓

Comment (tgreenwatts) [reply packages/table/preview/mutableTable.tsx]: Well if we do that, we should disable "Editing" with a note about this.

Comment (cmslewis) [reply packages/table/preview/mutableTable.tsx]: @gscshoyru you can make the editing toggle dependent on this one:

```tsx
{this.renderSwitch("Editing", "enableColumnNameEditing", "enableColumnCustomHeaders", false)}
``` 

Then the editing switch will be disabled and show a tooltip that says:
`Requires enableColumnCustomHeaders=false` on hover.

Comment (cmslewis) [reply packages/table/preview/mutableTable.tsx]: Oh, and also, `Sentence cased` please.

**Comment by themadcreator** [comment]
toggle-col-header
is this right?

**Comment by gscshoyru** [comment]
...that would be a no. 

Not related to this fix, obviously is bug in table that's been missed up until now. Please file.

**Comment by tgreenwatts** [comment]
Pretty sure that bug was fixed by Chris's empty stats PR @themadcreator

**Comment by cmslewis** [comment]
Yep just fixed that in #1663 @themadcreator.

**Pull request merged**